redisinsight/ui/src/pages/vector-search/components/commands-view/CommandsView/styles.module.scss

```diff
@@ -0,0 +1,43 @@
+.wrapper {
```

Comment:
_I noticed that this component is a replica of its counterpart in [Wokbench](https://github.com/redis/RedisInsight/blob/main/redisinsight/ui/src/pages/workbench/components/wb-results/WBResults/styles.module.scss), so this comment doesn’t apply to this specific case._ 

Question: However, I’m curious, do we have a general strategy for using styled components over SASS modules, or are we still not clear about the preferred way to style our custom components?